Frontend/ui-customer/src/Screens/ParcelDeliveryFlow/ParcelDeliveryScreen/ComponentConfig.purs

```diff
@@ -0,0 +1,150 @@
+      }
+  in
+    primaryButtonConfig'
+
+genericHeaderConfig :: ST.ParcelDeliveryScreenState -> GenericHeader.Config
+genericHeaderConfig state = let
+  config = GenericHeader.config
+  genericHeaderConfig' = config
+    {
+      height = WRAP_CONTENT
+    , width = WRAP_CONTENT
+    , background = Color.white900
+    , prefixImageConfig {
+        height = V 25
+      , width = V 25
+      , imageUrl = HU.fetchImage HU.FF_COMMON_ASSET "ny_ic_chevron_left"
+      , margin = (Margin 12 12 12 12)
+      }
+    , textConfig {
+        text = getString $ if state.data.currentStage == ST.DELIVERY_INSTRUCTIONS then DELIVERY else DELIVERY_DETAILS
+      , color = Color.black800
+      }
+    , suffixImageConfig {
+        visibility = GONE
+      }
+    , padding = (Padding 0 5 0 0)
+    }
+  in genericHeaderConfig'
+
+chooseVehicleConfig :: ST.ParcelDeliveryScreenState -> ChooseVehicle.Config
+chooseVehicleConfig state =
+  let
+    config = ChooseVehicle.config
+    selectedParcelEstimate = state.data.parcelQuoteList
+    chooseVehicleConfig' = config
+      { vehicleImage = HU.getVehicleVariantImage selectedParcelEstimate.vehicleVariant ST.RIGHT_VIEW
+      , isSelected = true
+      , vehicleVariant = selectedParcelEstimate.vehicleVariant
+      , vehicleType = selectedParcelEstimate.vehicleType
+      , capacity = selectedParcelEstimate.capacity
+      , price = selectedParcelEstimate.price
+      , isCheckBox = false
+      , isEnabled = true
+      , index = selectedParcelEstimate.index
+      , activeIndex = selectedParcelEstimate.activeIndex
+      , id = selectedParcelEstimate.id
+      , maxPrice = selectedParcelEstimate.maxPrice
+      , basePrice = selectedParcelEstimate.basePrice
+      , showInfo = selectedParcelEstimate.showInfo
+      , searchResultType = selectedParcelEstimate.searchResultType
+      , isBookingOption = false
+      , pickUpCharges = selectedParcelEstimate.pickUpCharges 
+      , layoutMargin = Margin 16 0 16 0
+      , tollCharge = selectedParcelEstimate.tollCharge
+      , serviceTierName = selectedParcelEstimate.serviceTierName
+      , serviceTierShortDesc = selectedParcelEstimate.serviceTierShortDesc
+      , airConditioned = selectedParcelEstimate.airConditioned
+      , extraFare = selectedParcelEstimate.extraFare
+      , fareInfoDescription = selectedParcelEstimate.fareInfoDescription
+      , isNightShift = selectedParcelEstimate.isNightShift
+      , nightChargeTill = selectedParcelEstimate.nightChargeTill
+      , nightChargeFrom = selectedParcelEstimate.nightChargeFrom
+      , driverAdditions = selectedParcelEstimate.driverAdditions
+      , validTill = selectedParcelEstimate.validTill
+      , hasTollCharges = selectedParcelEstimate.hasTollCharges
+      , hasParkingCharges = selectedParcelEstimate.hasParkingCharges
+      , singleVehicle = true
+      }
+  in chooseVehicleConfig'
+
+rateCardConfig :: ST.ParcelDeliveryScreenState -> RateCard.Config
+rateCardConfig state =
+  RateCard.config 
```

Comment:
can we reuse homescreen rateCardConfig ? we can move this logic to seperate module and reuse at both the places

Reply:
Thought of keeping it different from Home screen's rateCard Config because in a later requirement for `insurance`, `lost and found` use cases will be just for Parcel Delivery. (Rental and Intercity has a separate `rentalRateCardConfig` for these reasons)